django_project/lesson/forms/section.py

```diff
@@ -0,0 +1,54 @@
+# coding=utf-8
+"""Section forms."""
+
+from django.forms import ModelForm
+from django.utils.translation import ugettext_lazy as _
+
+from crispy_forms.helper import FormHelper
+from crispy_forms.layout import (
+    Layout,
+    Fieldset,
+    Field,
+    Submit,
+)
+
+from lesson.models.section import Section
+
+
+class SectionForm(ModelForm):
+    """Form for creating section"""
+
+    class Meta:
+        model = Section
+        fields = (
+            'section_number',
+            'name',
+            'notes'
+        )
+
+    def __init__(self, *args, **kwargs):
+        self.project = kwargs.pop('project')
+        self.helper = FormHelper()
+        layout = Layout(
+            Fieldset(
+                _('Section details'),
+                Field('section_number', css_class='form_control'),
```

Comment:
I dont think it makes sense to let the user set a section number - it should be calculated dynamically. The section order is intended to be mostyly invisible to the user and will determine the sort order for the sections (and numbers need not be sequential).

Reply:
Have a look at how @dimasciput did the sortable implementation for changelog categories for an example of what we are looking for here...

Reply:
I just checked the form for creating new category, it also has the sorting number
<img width="1258" alt="screen shot 2018-01-11 at 15 20 23" src="https://user-images.githubusercontent.com/1421861/34815180-02cf35e4-f6e3-11e7-84f3-758b08d8ab3c.png">

But, I understand by ordering (after seeing the ordering in http://changelog.qgis.org/en/inasafe/category/order/ )

Can we do it later? So that we can divide the work?

Reply:
yes fine to do later thanks